src/sanitize-argument.ts

```diff
@@ -24,7 +26,15 @@ export async function sanitize_all(argv: any) {
   // sanitizing speed
   sanitize_speed(_speed);
 
-  // sanitizing s3
+  // sanitizing custom flavour
+  if (argv.customFlavour) {
+    argv.customFlavour = sanitize_customFlavour(argv.customFlavour);
+    if (!argv.customFlavour) {
+      throw new Error('Custom Flavour not found');
+    }
```

Comment:
I believe this can be done in one line

Reply:
exceptions can't be included in inline conditions using `||`

Reply:
I really wonder about that... but ok